desktop/libs/hadoop/src/hadoop/fs/upload.py

```diff
@@ -88,16 +92,16 @@ def __init__(self, request, *args, **kwargs):
     self.totalfilesize = kwargs.get('qqtotalfilesize')
     self.file_name = kwargs.get('qqfilename')
     if self.file_name:
-      self.file_name = unicodedata.normalize('NFC', self.file_name) # Normalize unicode
+      self.file_name = unicodedata.normalize('NFC', self.file_name)  # Normalize unicode
     self.dest = kwargs.get('dest')
     self.file_name = kwargs.get('qqfilename')
-    if kwargs.get('filepath', None) != None:
+    if kwargs.get('filepath', None) is not None:
```

Comment:
Can simplify this like for some other review comment above. Something like `if kwargs.get('filepath')`?

Reply:
Yes, changed it.